### DISCUSSION TITLE
Problem with Redirecting all yetiforce to one domain?

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#17339] [General] [answered]
Hi, I've got a apache2 server with virutal host files, however it keeps redirecting other domains to yetiforce. I cannot figure it out any help on what might cause this is appreciated.

**Comment by mariuszkrzaczkowski** [comment]
look at this
https://doc.yetiforce.com/introduction/requirements/
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/tests/setup/apache

**Comment by MonsterITServices** [comment, marked as answer]
I finally found the route cause, the SSL certificate was not enabled on certbot. therefore it couldn't direct to the new client.
